Code/Legacy/CrySystem/LevelSystem/LevelSystem.h

```diff
@@ -92,6 +94,16 @@ class CLevelSystem
     bool LoadLevel(const char* levelName) override;
     void UnloadLevel() override;
     bool IsLevelLoaded() override { return m_bLevelLoaded; }
+
+    AZStd::string GetCurrentLevelName() override
+    {
+        if (m_pCurrentLevel && m_pCurrentLevel->GetLevelInfo())
+        {
+            return m_pCurrentLevel->GetLevelInfo()->GetName();
+        }
+        return "";
```

Comment:
returning "" is a pessimization.
A default constructed AZStd::string should be returned instead
```suggestion
        return AZStd::string{};
```

Reply:
Using const char* thanks :)